v1/python-sdk/tutorials/automl-with-azureml/forecasting-bike-share/auto-ml-forecasting-bike-share.ipynb

```diff
@@ -162,99 +163,148 @@
-   "source": [
-    "datastore = ws.get_default_datastore()\n",
-    "datastore.upload_files(\n",
-    "    files=[\"./bike-no.csv\"], target_path=\"dataset/\", overwrite=True, show_progress=True\n",
-    ")"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
     "Let's set up what we know about the dataset. \n",
     "\n",
     "**Target column** is what we want to forecast.\n",
     "\n",
     "**Time column** is the time axis along which to predict."
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "target_column_name = \"cnt\"\n",
     "time_column_name = \"date\""
-   ]
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {}
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "You are now ready to load the historical bike share data. We will load the CSV file into a plain pandas DataFrame."
+   ],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
-    "dataset = Dataset.Tabular.from_delimited_files(\n",
-    "    path=[(datastore, \"dataset/bike-no.csv\")]\n",
-    ").with_timestamp_columns(fine_grain_timestamp=time_column_name)\n",
+    "all_data = pd.read_csv(\"bike-no.csv\", parse_dates=[time_column_name])\n",
     "\n",
     "# Drop the columns 'casual' and 'registered' as these columns are a breakdown of the total and therefore a leak.\n",
-    "dataset = dataset.drop_columns(columns=[\"casual\", \"registered\"])\n",
-    "\n",
-    "dataset.take(5).to_pandas_dataframe().reset_index(drop=True)"
-   ]
+    "all_data.drop([\"casual\", \"registered\"], axis=1, inplace=True)"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": false,
+     "outputs_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
-    "### Split the data\n",
-    "\n",
+    "### Split the data\r\n",
+    "\r\n",
     "The first split we make is into train and test sets. Note we are splitting on time. Data before 9/1 will be used for training, and data after and including 9/1 will be used for testing."
-   ]
+   ],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "# select data that occurs before a specified date\n",
-    "train = dataset.time_before(datetime(2012, 8, 31), include_boundary=True)\n",
-    "train.to_pandas_dataframe().tail(5).reset_index(drop=True)"
-   ]
+    "train = all_data[all_data[time_column_name] <= pd.Timestamp(\"2012-08-31\")]\n",
```

Comment:
To avoid `SettingWithCopyWarning`, please copy the partitions:
"train = all_data[all_data[time_column_name] <= pd.Timestamp(\"2012-08-31\")].copy()\n",
"test = all_data[all_data[time_column_name] >= pd.Timestamp(\"2012-09-01\")].copy()"